R/append_values.R

```diff
@@ -7,7 +7,7 @@
 #'
 #' @export
 
-tri_append_bias <- function(dat, values, common = T) {
+tri_append_bias_modi <- function(dat, values, common = T) {
```

Comment:
I think we should leave the name the same, as this is an issue that will need to be resolved for all users of the package.